zdatasets/utils/aws.py

```diff
@@ -63,6 +60,46 @@ def get_aws_client(role_arn: str, service: str):
     return session.client(service)
 
 
+def send_sqs_message(
+    queue_url: str,
```

Comment:
do you have another PR that shows where the config will be input?

Reply:
yes, will have separate PRs in repo profile, webhook and kfp etc.